src/integration/pages/onDemandAudioPage/pashtoBrand/__snapshots__/canonical.test.js.snap

```diff
@@ -22,10 +22,10 @@ exports[`Canonical On Demand Audio Page Analytics ATI tracking pixel search para
     "s2": "68",
   },
   {
-    "p": "pashto.bbc_pashto_radio.w172xjcjkv7gw1w.page",
+    "p": "pashto.bbc_pashto_radio.programmes.p0340v0j.page",
```

Comment:
Does this mean that the test was previously wrong, and using an audio episode instead of a brand?

Reply:
But to me this change makes sense, because brands do have 'programmes' in their identifier.

Reply:
Yes, I think this also looks right to me, we're using the correct asset for this test